test/functional/api/cas/cache_config.py

```diff
@@ -110,5 +111,22 @@ def default_seq_cut_off_params():
 # TODO: Use case for this will be to iterate over configurations (kernel params such as
 # TODO: io scheduler, metadata layout) and prepare env before starting cache
 class CacheConfig:
-    def __init__(self):
-        pass
+    def __init__(self,
+                 cache_line_size=CacheLineSize.DEFAULT,
+                 cache_mode=CacheMode.DEFAULT,
+                 cleaning_policy=CleaningPolicy.DEFAULT,
+                 eviction_policy=EvictionPolicy.DEFAULT,
+                 metadata_mode=MetadataMode.normal):
+        self.cache_line_size = cache_line_size
+        self.cache_mode = cache_mode
+        self.cleaning_policy = cleaning_policy
+        self.eviction_policy = eviction_policy
+        self.metadata_mode = metadata_mode
+
+
+def validate_cache_params(cache, cache_config: CacheConfig):
```

Comment:
I think it would be clearer if we had implemented `__eq__()` in `CacheConfig`. Then we could just do:
```python
assert config == cache.get_cache_config()
```